Source/WebCore/platform/graphics/gstreamer/eme/WebKitOpenCDMWidevineDecryptorGStreamer.cpp

```diff
@@ -0,0 +1,222 @@
+/* GStreamer OpenCDM decryptor
+ *
+ * Copyright (C) 2016-2017 TATA ELXSI
+ * Copyright (C) 2016-2017 Metrological
+ *
```

Comment:
Since you used our code as template and reused a lot of them, it is necessary to respect that and add Igalia name to the copyright too. This applies to the .h too.